
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="google-site-verification" content="f_--N3WuBtfzUzdq05UpgO98Vy6YeBD9qdJ0v6geVBg" />
	<title>Kang Jin's Homepage</title>

    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- Le styles -->
    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
    <link href="assets/css/bootstrap-responsive.min.css" rel="stylesheet">
    <link href="assets/css/feixia.css" rel="stylesheet">

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="assets/js/html5shiv.js"></script>
    <![endif]-->
  </head>

  <body>

    <div class="container">
      <div class="row">
        <div class="span3 bs-docs-sidebar">
          <!-- We use a fancy nav bar if there is enough space -->
          <hr class="hidden-phone"/>
          <ul class="nav nav-list bs-docs-sidenav hidden-phone">
            <li><a href="#"><i class="icon-play"></i>Home</a></li>     
      <li><a href="#Research Interest"><i class="icon-play"></i>Research Interest</a></li>
			<li><a href="#position & education"><i class="icon-play"></i>Position & Education</a></li>
			<li><a href="#Publications"><i class="icon-play"></i>Publiacations</a></li>
            <li><a href="#softwares"><i class="icon-play"></i>Softwares </a></li>
            <li><a href="#resources"><i class="icon-play"></i>Resources </a></li>
			<li><a href="#honors"><i class="icon-play"></i>Honors</a></li>
            		<li><a href="assets/pdf/CV_KangJin_updating_new.pdf"><i class="icon-play"></i>CV</a></li>
			
		  </ul>
          <hr class="hidden-phone"/>
          <div class="text-center hidden-phone">
            <img src="assets/img/kang_175x20.jpg" alt="photo" class="logo-image"/>
          </div>
          <!-- Otherwise, we simply use a flat list of links -->
          <div class="visible-phone">
            <a href="#"><i class="icon-play"></i>Home</a>
		<a href="#education"><i class="icon-play"></i>Education</a>
            <a href="#manuscripts"><i class="icon-play"></i>Manuscripts</a>
            <a href="#publications"><i class="icon-play"></i>Publications</a>
            <a href="#honors"><i class="icon-play"></i>Honors</a>
		<li><a href="#softwares"><i class="icon-play"></i>Softwares </a></li>
            <a href="assets/pdf/CV_KangJin_updating_new.paper"><i class="icon-play"></i>CV</a>
          </div>
        </div>
        <div class="span9">
          <h3>
	Kang Jin
</h3>
<h5>
Kang.Jin at cchmc.org

<a href="https://github.com/KANG-BIOINFO">[Github]</a>
<a href="https://scholar.google.com/citations?user=nUFEZw4AAAAJ&hl=en">[Google Scholar]</a>
<a href="https://twitter.com/Kang_Jin__">[Twitter]</a>
</h5>
Last updated: 09/25/2023

<h3>Research Interest</h3>
<a name="Research Interest"></a>
<p>
    Hi, I am a fifth-year Ph.D. student in <a href="https://www.cincinnatichildrens.org/research/divisions/b/bmi">Biomedical Informatics Division</a> under the supervision of <a href="https://www.cincinnatichildrens.org/bio/a/bruce-aronow">Dr. Bruce Aronow</a> at <a href="https://www.cincinnatichildrens.org/">Cincinnati Children’s Hospital</a>. I am currently visiting <a href="https://www.jianshulab.org/team">Shu lab</a> at Massachusetts General Hospital, where I am learning spatial transcriptomics technologies.<br><br>
    
    As an enthusiastic researcher, I am passionate about harnessing advanced computational techniques to unveil the biological mechanisms hidden within <b>high-throughput sequencing</b> and <b>high-content imaging</b> data, with a specific focus on <b>image-based spatial profiling</b>.
     My present work primarily centers around the development of cutting-edge <b>deep learning models</b> tailored to image-based spatial omics data <i><a href='https://www.biorxiv.org/content/10.1101/2023.09.19.558548v1'>(Jin*, Zhang* et al., bioRxiv, 2023)</a></i> 
     and the analysis of <b>temporal perturbation patterns</b> in single-cell experiments <i><a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac324/6673850">(Jin et al., Briefings in Bioinformatics, 2022)</a></i>. 
     My commitment extends to active involvement in the experimental design of spatial multi-omics experiments, 
     and I possess a wealth of experience in exploring single cell data across various biological systems and diseases, 
     including HLHS <i><a href="https://doi.org/10.1016/j.stem.2022.03.003">(Xu, Jin et al., Cell Stem Cell, 2022)</a></i>, 
     schizophrenia <i><a href="https://www.nature.com/articles/s41467-023-39420-6">(Sebastian*, Jin* et al. Nat Communications, 2023)</a></i>, among others.<br><br>
    
    I am particularly interested in cutting-edge bioengineering innovations, such as <b>image-based spatial transcriptomics</b> and <b>optical molecular recording</b>. 
    Collaborating with <a href="https://www.jianshulab.org/team">Dr. Jian Shu</a>, we are devotedly developing innovative methods for in-situ molecular recording through microscopy. <br><br>
    
    With expertise in computational biology and a strong interest in bioengineering, 
    my ultimate goal is to advance our understanding of neuroscience and cancer biology by implementing cutting-edge computational methods with emerging new molecular recording technologies. <br><br>
        
<p>

<div>


<h3>Position & Education</h3>
<a name="position & education"></a>
<div class="media">
  <span class="pull-left"><img src="./assets/img/mgh_logo.png" width="80px" height="96px"/></span>
  <div class="media-body">
      <p><span style="font-weight: bold">2022 -  Present</span>, Cutaneous Biology Research Center, <i><b>Massachusetts General Hospital, Harvard Medical School</b></i>,</p>
      <p> Visiting student</p>
  </div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/cchmc_logo.jpeg" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2018 -  Present</span>, Department of Biomedical Informatics, <i><b>Cincinnati Children's Hospital, University of Cincinnati</b></i>,</p>
        <p> Ph.D. student</p>
    </div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/zju_logo.png" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2014 - 2018</span>, Department of Life Sciences, <i><b>Zhejiang University</b></i>,</p>
        <p>Bachelor of Biological Science </p>
    </div>
</div>
		
		
<h3>Publications</h3>
<a name="Publications"></a>
(*Equal contribution)<br>

<b>Published papers (as main author)</b><br>
<ul>
  <li><p><font color="#0f5ddb"> Bering: joint cell segmentation and annotation for spatial transcriptomics with transferred graph embeddings </font> 
<a href="https://www.biorxiv.org/content/10.1101/2023.09.19.558548v1">[paper]</a> <a href="https://github.com/jian-shu-lab/Bering">[code]</a> <br>
<b>Kang Jin*</b>, Zuobai Zhang*, Ke Zhang, Francesca Viggiani, Claire Callahan, Jian Tang, Bruce J. Aronow, Jian Shu<br>
<i> <font color="#0000ff"> bioRxiv </font>(2023).</I>
</p></li> 
</ul>

<ul>
  <li><p><font color="#0f5ddb"> CellDrift: Inferring Perturbation Responses in Temporally-Sampled Single Cell Data </font> 
<a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac324/6673850">[paper]</a> <a href="https://github.com/KANG-BIOINFO/CellDrift">[code]</a> <br>
<b>Kang Jin</b>, Dan Schnell, Guangyuan Li, Nathan Salomonis, Surya Prasath, Rhonda Szczesniak, Bruce J Aronow<br>
<i> <font color="#0000ff"> Briefings in Bioinformatics </font>(2022).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> Single cell transcriptomic profiling of human brain organoids reveals developmental timing- and cell-type-specific vulnerabilities induced by NRXN1 CNVs in schizophrenia </font> 
<a href="https://www.nature.com/articles/s41467-023-39420-6">[paper]</a> <a href="https://github.com/KANG-BIOINFO/NRXN1_effects">[code]</a> <br>
Rebecca Sebastian*, <b>Kang Jin*</b>, Narciso Pavon, Ruby Bansal, Andrew Potter, Yoonjae Song, Juliana Babu, Rafael Gabriel, Yubing Sun, Bruce Aronow, ChangHui Pak<br>
<i> <font color="#0000ff"> Nature Communications </font>(2023).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> An interactive single cell web portal identifies gene and cell networks in COVID-19 host responses </font> 
<a href="https://reader.elsevier.com/reader/sd/pii/S258900422101083X?token=132EAD4906F46833416A280EE4F84B37125CA5F3753F9FC03CBFAD03BD06BBE9E986DC4D82A99A4184F609DB77393389&originRegion=us-east-1&originCreation=20211019022600">[paper]</a> <a href="https://github.com/KANG-BIOINFO/COVID-19-Atlas">[code]</a> <br>
<b>Kang Jin</b>, Eric E. Bardes, Alexis Mitelpunkt, Jake Y. Wang, Surbhi Bhatnagar, Soma Sengupta, Daniel Pomeranz Krummel, Marc E. Rothenberg, Bruce J. Aronow<br>
<i> <font color="#0000ff"> iScience </font>(2021).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> Uncompensated mitochondrial oxidative stress underlies heart failure in an iPSC-derived model of congenital heart disease </font> 
<a href="https://doi.org/10.1016/j.stem.2022.03.003">[paper]</a> <br>
Xinxiu Xu, <b>Kang Jin</b>, Abha S Bais, Wenjuan Zhu, Hisato Yagi, Timothy N Feinstein, Phong K Nguyen, Joseph D Criscione, Xiaoqin Liu, Gisela Beutner, Kalyani B Karunakaran, Krithika S Rao, Haoting He, Phillip Adams, Catherine K Kuo, Dennis Kostka, Gloria S Pryhuber, Sruti Shiva, Madhavi K Ganapathiraju, George A Porter Jr, Jiuann-Huey Ivy Lin, Bruce Aronow, Cecilia W Lo<br>
<i> <font color="#0000ff"> Cell Stem Cell </font>(2022).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> Identification of distinct tumor cell populations and key genetic mechanisms through single cell sequencing in hepatoblastoma </font> 
<a href="https://www.nature.com/articles/s42003-021-02562-8">[paper]</a> <a href="https://github.com/KANG-BIOINFO/scRNA-seq_Hepatoblastoma">[code]</a> <br>
Alexander Bondoc*, Kathryn Glaser*, <b>Kang Jin*</b>, Charissa Lake, Stefano Cairo, James Geller, Gregory Tiao, Bruce Aronow<br>
<i> <font color="#0000ff"> Communications Biology </font>(2021).</I>
</p></li> 
</ul>


<b>Published papers (as contributing author)</b><br>
<ul>
  <li><p><font color="#0f5ddb"> Guided construction of single cell reference for human and mouse lung </font> 
<a href="https://www.nature.com/articles/s41467-023-40173-5">[paper]</a> <br>
Minzhe Guo, Michael P Morley, Yixin Wu, Yina Du, Shuyang Zhao, Andrew Wagner, Michal Kouril, <b>Kang Jin</b>, ..., Yan Xu<br>
<i> <font color="#0000ff"> Nature Communications </font>(2023).</I>
</p></li> 
</ul>

<ul>
  <li><p><font color="#0f5ddb"> KAT5 activity regulates G0-like states in human gliomas </font> 
<a href="https://www.biorxiv.org/content/10.1101/2022.03.17.484768v2.abstract">[paper]</a> <br>
Anca B. Mihalas, Sonali Arora, Samantha A. O’Connor, Heather M. Feldman, John Bassett, Kelly Mitchell, <b>Kang Jin</b>, ..., Patrick J. Paddisons<br>
<i> <font color="#0000ff"> bioRxiv </font>(2022).</I>
</p></li> 
</ul>

<ul>
  <li><p><font color="#0f5ddb"> LungMAP Portal Ecosystem: Systems-Level Exploration of the Lung </font> 
<a href="https://www.atsjournals.org/doi/abs/10.1165/rcmb.2022-0165OC">[paper]</a>  <br>
Nathan Gaddis, Joshua Fortriede, Minzhe Guo, ..., <b>Kang Jin</b>, ..., Nathan Salomonis, NHLBI LungMAP Consortium<br>
<i> <font color="#0000ff"> American Journal of Respiratory Cell and Molecular Biology </font>(2022).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> Loss of Neurodevelopmental Gene CASK Disrupts Neural Connectivity in Human Cortical Excitatory Neurons </font> 
<a href="https://www.sciencedirect.com/science/article/pii/S2589004222014596">[paper]</a> <br>
Danny McSweeney, Rafael Gabriel, <b>Kang Jin</b>, Zhiping P Pang, Bruce J Aronow, ChangHui Pak<br>
<i> <font color="#0000ff"> iScience </font>(2022).</I>
</p></li> 
</ul>
<ul>
  <li><p><font color="#0f5ddb"> A census of the lung: CellCards from LungMAP </font> 
<a href="https://www.sciencedirect.com/science/article/pii/S1534580721008923">[paper]</a> <br>
Xin Sun, Anne-Karina Perl, Rongbo Li, ..., <b>Kang Jin</b>, ..., Edward E Morrisey, NHLBI LungMAP Consortium <br>
<i> <font color="#0000ff"> Developmental Cell </font>(2022).</I>
</p></li> 
</ul>
<ul>
    <li><p><font color="#0f5ddb"> Cross-platform validation of neurotransmitter release impairments in schizophrenia patient-derived NRXN1-mutant neurons </font> 
	<a href="https://www.pnas.org/content/118/22/e2025598118.short">[paper]</a> <br>
	ChangHui Pak,  Tamas Danko, Vincent R. Mirabella, Jinzhao Wang, Yingfei Liu, Madhuri Vangipuram, Sarah Grieder, Xianglong Zhang, Thomas Ward, Yu-Wen Alvin Huang, <b>Kang Jin</b>, ..., Thomas C. Südhof<br>
	<i> <font color="#0000ff"> PNAS </font>(2021).</I>
	</p></li> 
</ul>

<ul>
  <li><p><font color="#0f5ddb"> Functional immune deficiency syndrome via intestinal infection in COVID-19 </font> 
<a href="https://www.biorxiv.org/content/10.1101/2020.04.06.028712v2.full">[paper]</a> <br>
Erica T. Prates, Michael R. Garvin, Mirko Pavicic, Piet Jones, Manesh Shah, Christiane Alvarez, David Kainer, Omar Demerdash, B Kirtley Amos, Armin Geiger, John Pestian, <b>Kang Jin</b>, Alexis Mitelpunkt, Eric Bardes, Bruce Aronow, Daniel Jacobson<br>
<i> <font color="#0000ff"> bioRxiv </font>(2020).</I>
</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Predicted Arabidopsis interactome resource and gene set linkage analysis: a transcriptomic analysis resource </font> 
	<a href="https://academic.oup.com/plphys/article/177/1/422/6117111?login=true">[paper]</a> <br>
	Heng Yao, Xiaoxuan Wang, Pengcheng Chen, Ling Hai, <b>Kang Jin</b>, Lixia Yao, Chuanzao Mao, Xin Chen<br>
	<i> <font color="#0000ff"> Plant Physiology </font>(2018).</I>
	</p></li> 
</ul>


<h3>Softwares</h3>
<a name="softwares"></a>

<ul>
  <li><p><a href="https://github.com/jian-shu-lab/Bering">Bering</a>: Cell segmentation and annotation in high-resolution spatial transcriptomics data
</p></li> 
</ul>

<ul>
    <li><p><a href="https://toppcell.cchmc.org/biosystems/go/index3/">ToppCell</a>: Single-cell gene module exploratory platform
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/KANG-BIOINFO/ToppCellPy">ToppCellPy</a>: Python package for ToppCell. <a href="https://nbviewer.org/github/KANG-BIOINFO/ToppCellPy/blob/main/test/COVID-19%20example.ipynb">Tutorial </a>on COVID-19 example.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/KANG-BIOINFO/CellDrift">CellDrift</a>: Inferring Perturbation Responses in Temporally-Sampled Single Cell Data.
	</p></li> 
</ul>

<ul>
  <li><p><a href="https://github.com/KANG-BIOINFO/Bering">Bering</a>: Gene colocalization-aware segmentation of image-based spatial transcriptomics using deep learning models (Wrapping up)
</p></li> 
</ul>

<h3>Resources</h3>
<a name="resources"></a>
<ul>
    <li><p><a href="https://drive.google.com/file/d/1_10RAY-M4gxndvJmUhjzSFUjpeMPoQop/view?usp=sharing"> Bering Poster</a><br>
      Poster of Bering on <a href="https://sites.google.com/cs.washington.edu/mlcb2022/">MLCB 2022</a> 
    <li><p><a href="https://drive.google.com/file/d/1pUPgXRQz3jQqlefbHa0eubCr5b_uOIMp/view?usp=sharing">CellDrift Poster</a><br> Poster of CellDrift on <a href="https://www.iscb.org/ismb2022">ISMB 2022 (Sysmod)</a>.
      Second Place, Best Poster Award.
    <li><p><a href="https://drive.google.com/file/d/1EweBgl4RWNdxAASz5L-6FbnVCc31fZpy/view?usp=sharing">CellDrift Poster</a><br> Poster of CellDrift on <a href="https://probgen22.github.io/">ProbGen22 conference</a>.
    </p></li> 
    <li><p><a href="https://toppcell.cchmc.org/biosystems/go/index3/COVID-19%20Atlas">COVID-19 Gene Signature Atlas</a><br>
        Modules of most significant gene signatures for COVID-19, which were derived from more than 10 public datasets. More details can be found <a href="https://www.sciencedirect.com/science/article/pii/S258900422101083X">here</a>.
    </p></li> 
    <li><p><a href="https://drive.google.com/file/d/1HnP_Q7PEuOoSNR--Kr7THjcrPI2Ie6S0/view?usp=sharing">Digestion of BICCN Nature Paper Collections</a><br>
        A brift digestion of 16 BICCN papers published on Nature on Oct 2021, which introduced comprensive brain atlases and resources covering transcriptomic, epigenomic, morphogenic, eletrophysiological, spatial and other data.
    </p></li> 
    <li><p><a href="https://drive.google.com/file/d/1ZzLbmGgDBhmIxSWtu1vICmZy8DBXPdSI/view?usp=sharing">Journal Club: The specious Art of Single Cell Genomics</a><br>
        Slide for a journal club, which covered an overview of current neighbor embedding methods and discussion about Dr. Lior Pachter's recent preprint on UMAP.
	</p></li> 
</ul>

<h3>Honors and Awards</h3>
<a name="honors"></a>
<ul>
<li> Second Place, Best Poster Award, ISMB 2022 (SysMod Meeting) 2022 </li>
<li> Second Place in Graduate Student Research Forum (GSRF), University of Cincinnati, 2020 </li>
<li> Best Application Award of Zhejiang Bioinformatics Competition,2017 </li>
<li> China National Award, 2017 </li>
<li> First Prize Scholarship for Excellent Students, Zhejiang University, 2017 </li>
<li> Tang Lixin Scholarship for Excellent Model Student, 2017 </li>
<li> Zhejiang Provincial Government Scholarship, 2016 </li>
<li> Second Prize, Physics Innovation Competition in Zhejiang Province, 2015 </li>
</ul>



        </div>
      </div>
    </div>

    <!-- Footer
    ================================================== -->
    <hr>
    <footer class="footer">
    <div class="container">
      <div class="row">
        <div class="span12">
          <p>&copy; Kang Jin. Inspired by <a href="https://martinjzhang.github.io/">Martin's</a>.
		  </p>
        </div>
      </div>
    </div>
    </footer>



    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
<!--
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
    <script src="assets/js/bootstrap-transition.js"></script>
    <script src="assets/js/bootstrap-alert.js"></script>
    <script src="assets/js/bootstrap-modal.js"></script>
    <script src="assets/js/bootstrap-dropdown.js"></script>
    <script src="assets/js/bootstrap-scrollspy.js"></script>
    <script src="assets/js/bootstrap-tab.js"></script>
    <script src="assets/js/bootstrap-tooltip.js"></script>
    <script src="assets/js/bootstrap-popover.js"></script>
    <script src="assets/js/bootstrap-button.js"></script>
    <script src="assets/js/bootstrap-collapse.js"></script>
    <script src="assets/js/bootstrap-carousel.js"></script>
    <script src="assets/js/bootstrap-typeahead.js"></script>
    <script src="assets/js/bootstrap-affix.js"></script>
    <script src="assets/js/holder/holder.js"></script>
    <script src="assets/js/application.js"></script>
-->
  </body>
</html>
